crates/l2/storage/src/error.rs

```diff
@@ -0,0 +1,59 @@
+use ethrex_rlp::error::RLPDecodeError;
+use ethrex_trie::TrieError;
+#[cfg(feature = "redb")]
+use redb::{CommitError, DatabaseError, StorageError, TableError, TransactionError};
+use thiserror::Error;
+
+// TODO improve errors
+#[derive(Debug, Error)]
+pub enum RollupStoreError {
+    #[error("DecodeError")]
+    DecodeError,
+    #[cfg(feature = "libmdbx")]
```

Comment:
Would it be too hard to have a simple "DBError" and try to add the DB name to the error's string instead of having a variant for each flag?